addons/payment/tests/test_flows.py

```diff
@@ -376,3 +377,11 @@ def test_payment_by_token_triggers_exactly_one_payment_request(self):
                 **self._prepare_transaction_values(self.create_token().id, 'token')
             )
             self.assertEqual(patched.call_count, 1)
+
+    def test_delete_apml_from_journal_when_active_provider(self):
```

Comment:
This would rather fit in a new `test_account_journal.py` file since it tests the behavior of that model. `test_flows.py` is only for payment flows.